phpunit.xml.dist

```diff
@@ -11,15 +11,8 @@
          beStrictAboutOutputDuringTests="true"
          beStrictAboutChangesToGlobalState="true"
          checkForUnintentionallyCoveredCode="false"
+         printerClass="\Drupal\Tests\Listeners\HtmlOutputPrinter"
```

Comment:
I love this! Everytime I'm adding this manually :)

Reply:
Yeah it turns out this bug in PHPStorm that was originally causing it to be commented out was already fixed years ago, but the issue was never set to "fixed" :grin: